Automattic/woocommerce-payments
# Update new onboarding flow components to use admin color schema

Fixes #7098 

#### Changes proposed in this Pull Request
- Update the following components to use admin schema colors:
  - `RadioCard` – make it also usable by keyboard.
  - `LoadBar`
  - The onboarding stepper progress bar at the top.

I've also checked if it was possible to replace any of those components with `@wordpress/components` or `@woocommerce/components`, but unfortunately the only matching ones are experimental or not externally usable for now. We could check it again in the future, or contribute to make them usable from WP/WC.

#### Testing instructions
- Go to **Users → Profile** and pick a different admin color schema. (e.g. Midnight)
- Go to the new onboarding flow.
- Ensure that the mentioned components use the admin color schema. You can check the following screenshots for reference.

#### Before vs After
<img src="https://github.com/Automattic/woocommerce-payments/assets/7670276/fe4d0b10-566a-41f3-bff5-3474a81f7b76" width="507" />
<img src="https://github.com/Automattic/woocommerce-payments/assets/7670276/f3908168-d2e7-49db-b402-99ec30403f5b" width="750" />

-------------------


**Post merge**
